multiclient/client.go

```diff
@@ -520,6 +538,56 @@ func (mc *Client) BatchCallContext(ctx context.Context, isPostToAll bool, b []rp
 	return nil
 }
 
+// Subscribe API
+
+// SubscribeNewHead subscribes to notifications about the current blockchain head
+// on the given channel.
+func (mc *Client) SubscribeNewHead(ctx context.Context, ch chan<- *types.Header) (ethereum.Subscription, error) {
+	clientsMap := mc.rpcClientMap.Map()
+	lens := len(clientsMap)
+	if lens == 0 {
```

Comment:
It's a good check before perform rpc call.
It's better to add check for each method.

Reply:
OK